support/Pipelines/Bbh/Ringdown.py

```diff
@@ -229,11 +235,12 @@ def start_ringdown(
         ahc_dt2_datfile.append(ringdown_ylm_coefs[2])
     logger.info("Obtained ringdown coefs")
     # Print out coefficients for insertion into BBH domain
-    logger.info("Expansion: " + str(expansion_func_with_2_derivs))
+    logger.info("Expansion: " + str(evaluated_fot_dict["Expansion"]))
```

Comment:
are all of these logging calls really needed? If they're for debugging then make them `logger.debug`, or delete because all params are printed out anyway when an error occurs.

Reply:
I'll change them to logger.debug